cmd/influx/v1_dbrps.go

```diff
@@ -0,0 +1,180 @@
+package main
```

Comment:
Nitpick: I'd have kept the file name as `v1_dbrp` to match the command name 1:1

Reply:
Changed it back to `v1_dbrp` to match the command name! I was aiming for consistency but there's a mix of singular and plural no matter what so matching the command name seems best!